test/parallel/test-tls-ticket-cluster.js

```diff
@@ -46,7 +46,9 @@ if (cluster.isPrimary) {
       session: lastSession,
       rejectUnauthorized: false
     }, () => {
-      c.end();
+      c.on('end', () => {
+        c.end();
+      });
```

Comment:
Just a nit that can be ignored.

```suggestion
      c.on('end', c.end);
```